ts/test/checker-test.ts

```diff
@@ -19,7 +19,7 @@ import {PackageJson} from '../src/package-json-file';
 
 import proxyquire = require('proxyquire');
 
-proxyquire('../src/checker', {
+const checker = proxyquire('../src/checker', {
```

Comment:
Why is the `checker` variable needed here?

Reply:
It's used from line 59 below. I believe that is how `proxyquire` is supposed to be used. See examples in https://github.com/thlorenz/proxyquire. You should use `proxyquire` as you would use `require`.

Reply:
Ok sorry.  I missed that.  Yes, IIUC this is how proxyquire should be used.